<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogVKeyboard</class>
 <widget class="QDialog" name="DialogVKeyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>251</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="keylistWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>121</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SendPB">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Keys</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
